.github/workflows/linux-nightly-install.yml

```diff
@@ -30,6 +29,8 @@ jobs:
 
       - name: Setup development environment
         uses: ./.github/actions/setup-dev-env
+        with:
+          python-version: ${{ matrix.python-version }}
```

Comment:
Without this we are only going to test the default version, which is 3.7 at this point